remotes/push.go

```diff
@@ -130,3 +130,21 @@ func pushPayload(ctx context.Context, resolver remotes.Resolver, reference strin
 	}
 	return err
 }
+
+func pushBundleConfig(ctx context.Context, resolver remotes.Resolver, reference string, bundleConfig *converter.PreparedBundleConfig, allowFallbacks bool) (ocischemav1.Descriptor, error) {
```

Comment:
This can be a `PreparedBundleConfig` method, I guess it can clean some code.

Reply:
not the same package (don't want to add dependencies to the converter package)